### PR TITLE
Add SE2_002 Welcome msg & list

### PR DESCRIPTION
**Pull request by hyl246**
The list of actions will be updated when more commands are developed.

### CODE REVIEW

File: newbank/server/NewBankClientHandler.java [hunk 1]
```diff
@@ -33,12 +33,30 @@ public void run() {
...
 					String request = in.readLine();
+					if(request.equals("QUIT")){
+						out.println("*  Thank you for using NewBank  *\n Have a good day");
+						System.exit(0);
+					}
 					System.out.println("Request from " + customer.getKey());
 					String responce = bank.processRequest(customer, request);
 					out.println(responce);
+					out.println("Continue to other service, select an option and type a command: \n");
+					out.println("1. SHOWMYACCOUNTS - Show my accounts");
```
Comment (ph691) [inline newbank/server/NewBankClientHandler.java:52]: There is a bit of duplication here, we could move some of this into methods to reduce it.

**Review by ph691 — CHANGES_REQUESTED** [review]
Just a small comment to address.

**Comment by ph691** [comment]
@hyl246 can you also please resolve the merge conflicts on this branch? Just need to merge the main branch into this one locally and push up the commit

File: newbank/server/NewBankClientHandler.java (hunk 1 above)
Comment (hyl246) [reply newbank/server/NewBankClientHandler.java]: commands method is added

**Comment by hyl246** [comment]
conflicts should be resolved. Please check again thanks.

**Review by ph691 — APPROVED** [review]

**Comment by ph691** [comment]
Thanks for making those changes and fixing the merge conflicts. Looks good now :)

**Pull request merged**